hbw/app/javascript/packs/hbw/components/form/select.js.jsx

```diff
@@ -99,29 +99,17 @@ modulejs.define('HBWFormSelect',
       onFormSubmit = () => {
         const el = this.select;
 
-        if (this.isValid()) {
+        if (this.props.isValid()) {
```

Comment:
You have `getValidationState` in props now. It returns exactly this value.
I think you should either use `this.props.isValid()` and remove `getValidationState` completely or use `getValidationState` in all places :)